datalake/dlfile.py

```diff
@@ -138,7 +137,7 @@ def _validate_bundle(bundle_filename):
 
     @staticmethod
     def _validate_bundle_version(bundle):
-        v = File._get_content_from_bundle(bundle, 'version')
+        v = File._get_content_from_bundle(bundle, 'version').decode('UTF-8')
```

Comment:
Ah yes. I'll change to lowercase for consistency. I did this because the `locale.getpreferredencoding` (or whatever it's called) on my system returns "UTF-8".